tests/e2e/standalone/uninstall_test.go

```diff
@@ -0,0 +1,72 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package standalone_test
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	dockerClient "github.com/docker/docker/client"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestStandaloneUninstall(t *testing.T) {
+	ensureDaprInstallation(t)
+
+	output, err := cmdUninstall()
+	t.Log(output)
+	require.NoError(t, err, "dapr uninstall failed")
+	assert.Contains(t, output, "Dapr has been removed successfully")
+
+	// verify that .dapr directory does not exist.
+	homeDir, err := os.UserHomeDir()
+	require.NoError(t, err, "failed to get user home directory")
+
+	daprPath := filepath.Join(homeDir, ".dapr")
+	require.NoDirExists(t, daprPath, "Directory %s does not exist", daprPath)
+
+	verifyNoContainers(t)
```

Comment:
how does this run successfully in windows?

Reply:
Interesting, have put this test behind slim mode check.